### PR TITLE
Add gen from ddl subcommand

### PR DESCRIPTION
**Pull request by mizkei**
Generate Go code from DDL file.

This subcommand generates Go code without spanner instance.

Add testdata whose column has OPTIONS.
`cloud.google.com/go/spanner/spansql` can't parse such columns.

### CODE REVIEW

File: test/testdata/schema.sql [hunk 1]
```diff
@@ -20,6 +20,8 @@ CREATE TABLE FullTypes (
   FTBytes BYTES(32) NOT NULL,
   FTBytesNull BYTES(32),
   FTTimestamp TIMESTAMP NOT NULL,
+  FTCommitTimestampNull TIMESTAMP OPTIONS (allow_commit_timestamp=true),
```
Comment (kazegusuri) [inline test/testdata/schema.sql]: ahhh! I changed tests to use a fake spanner server, so this schema does not support this DDL for now. I know this PR but I didn't see this change. I think these columns not affect generated codes, so how about just removing this change for now?
I want the official fake spanner to be able to parse this DDL actually even if it does not understand semantically.

File: cmd/gen_from_ddl.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	"github.com/spf13/cobra"
+	"go.mercari.io/yo/generator"
+	"go.mercari.io/yo/internal"
+	"go.mercari.io/yo/loaders"
+)
+
+var (
+	genFromDDLOpts = internal.ArgType{}
+	genFromDDLCmd  = &cobra.Command{
+		Use:   "gen-from-ddl",
```
Comment (kazegusuri) [inline cmd/gen_from_ddl.go]: The purpose of this command and the default command is same. I think it is better to be the same command.

How do you think to do like this?

* The command name is `generate` for both with using spanner and file.
* For the generation with spanner, it requires 3 arguments and it's the default behavior.
* With DDL, it requires a flag --from-ddl and an argument as the file path.

I'm not sure it's not confusing, but I want to consider to use the same command for both.

File: test/testdata/schema.sql (hunk 1 above)
Comment (mizkei) [reply test/testdata/schema.sql]: `cloud.google.com/go/spanner/spansql` can't parse such columns.
If testdata doesn't include these columns, codes written with `cloud.google.com/go/spanner/spansql` would pass tests.
So, I added these columns for cases that fail in using `cloud.google.com/go/spanner/spansql`.

File: cmd/gen_from_ddl.go (hunk 2 above)
Comment (mizkei) [reply cmd/gen_from_ddl.go]: OK.
I will rename subcommand name and add codes for using spanner instance.

**Comment by kazegusuri** [comment]
Sorry for late reply. Let's merge this.  Please squash commits

**Comment by kazegusuri** [comment]
@mizkei

**Comment by mizkei** [comment]
@kazegusuri 

I'm so sorry for too late. I missed notification.

**Pull request merged**